charts/airbyte/templates/_helpers.tpl

```diff
@@ -70,25 +70,21 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-
 {{/*
-Get the Postgresql credentials secret.
+Get the Postgresql credentials secret name.
 */}}
-{{- define "airbyte.postgresql.secretName" -}}
-{{- if and (.Values.postgresql.enabled) (not .Values.postgresql.existingSecret) -}}
-    {{- printf "%s" (include "airbyte.postgresql.fullname" .) -}}
-{{- else if and (.Values.postgresql.enabled) (.Values.postgresql.existingSecret) -}}
-    {{- printf "%s" .Values.postgresql.existingSecret -}}
+{{- define "airbyte.database.secret.name" -}}
+{{- if .Values.postgresql.enabled -}}
+    {{ template "postgresql.secretName" .Subcharts.postgresql }}
 {{- else }}
     {{- if .Values.externalDatabase.existingSecret -}}
         {{- printf "%s" .Values.externalDatabase.existingSecret -}}
     {{- else -}}
-        {{ printf "%s-%s" .Release.Name "externaldb" }}
+        {{ printf "%s-%s" (include "common.names.fullname" .) "secrets" }}
```

Comment:
I'm not too familiar with this template helper - how is this set?

Reply:
This is being included as a chart dependency https://github.com/airbytehq/airbyte/blob/master/charts/airbyte/Chart.yaml#L27-L30

It's not something I initially added but as it was there I've standardised it's use across this chart when it comes to resource names. You can see its use in other charts' helpers such as bitnami postgres https://github.com/bitnami/charts/blob/master/bitnami/postgresql/templates/_helpers.tpl#L11

Reply:
Got it. Makes sense. Thanks!